### DISCUSSION TITLE
Cloudflare Tunnel Hosted Website

### DISCUSSION THREAD

**Discussion by Mauzy0x00** [thinkst/canarytokens#346] [Q&A] [answered]
I have an Ubuntu server hosting my website and a few apps. I thought I'd give it a try hosting this with docker. The note says to point my nameservers to my docker image and my public IP. being that my site is hosted on 127.0.0.1 through a cloudflared tunnel and my nameservers are handled by cloudflare, I am unsure how I should configure my frontend.env and switchboard.env. Do I host this on a different subdomain? That is what I did for self-hosting my obsidian notes; Different subdomain and port but still through local host. 

Perhaps this is a silly question as I am still learning about networking. Any feedback or links to documentation would be greatly appreciated! :)

**Comment by roberto-aldera** [comment, marked as answer]
Hi @Mauzy0x00, sorry for the delay here - are you still having trouble with this? Did the [readme](https://github.com/thinkst/canarytokens?tab=readme-ov-file#configuration) help?

**Reply by Mauzy0x00** [reply]
Hello! I forgot about this -- thank you for the reply. 

I'll revisit setting this up tonight; I've learned a thing or two since asking this.
